### ISSUE TITLE
add multiple trackers at a time

### ISSUE THREAD

**Issue by naparlesc** [rakshasa/rtorrent#836]
I have a tracker list, but i dont know how to use it. Its possible to add the .txt file to a torrent ?

**Comment by chros73** [comment]
No, it's not possible. But take a look [at this](https://github.com/rakshasa/rtorrent/issues/695).
